packages/utils/telemetry-utils/src/staging.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ILoggingError } from "@fluidframework/common-definitions";
+
+// ///////////////////////////////////////////////////////////////////////// //
+//        THIS CODE TO BE MOVED TO COMMON-DEFINITIONS AND COMMON-UTILS       //
+// ///////////////////////////////////////////////////////////////////////// //
```

Comment:
why should this be in common?

Reply:
I want `IFluidErrorBase` to be accessible from anywhere in the codebase.  e.g. The error types in Container-Definitions would one day hopefully bump up to use this type.

Reply:
And now the only other stuff is type guards  which is pretty harmless to add to common-utils.

Reply:
in which code base. common should primarily be used for things shared between client and server. i think most our logging code shouldn't be there.

Reply:
Hmm ok.  Want me to get rid of this file in this PR?  I still have my eye on updating `assert` to yield an `IFluidErrorBase`, but at the same time, I see your point about server so... not sure.